src/Utils/request/utils.ts

```diff
@@ -96,3 +96,21 @@ export function handleUploadPercentage(
     setUploadPercent(percentComplete);
   }
 }
+
+export function MoveUp<T>(arr: T[], idx: number): T[] | undefined {
```

Comment:
follow proper js naming conventions!

```suggestion
export function moveUp<T>(arr: T[], idx: number): T[] | undefined {
```

Reply:
@rithviknishad addressed change and also switched question move up/down also using the functions in the util